pkg/kubelet/cm/container_manager_windows.go

```diff
@@ -84,13 +95,32 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 	if err != nil {
 		return nil, err
 	}
+	// Correct NUMA information is currently missing from cadvisor's
+	// MachineInfo struct, so we use the CPUManager's internal logic for
+	// gathering NUMANodeInfo to pass to components that care about it.
+	numaNodeInfo, err := cputopology.GetNUMANodeInfo()
+	if err != nil {
+		return nil, err
+	}
 	capacity = cadvisor.CapacityFromMachineInfo(machineInfo)
 
-	return &containerManagerImpl{
+	cm := &containerManagerImpl{
 		capacity:          capacity,
 		nodeConfig:        nodeConfig,
 		cadvisorInterface: cadvisorInterface,
-	}, nil
+	}
+
+	klog.Infof("Creating device plugin manager: %t", devicePluginEnabled)
+	if devicePluginEnabled {
+		cm.deviceManager, err = devicemanager.NewManagerImpl(numaNodeInfo, cm.topologyManager)
```

Comment:
Any ideas/pointers to how exactly `numaNodeInfo` is used by `devicemanager`? I guess this Q is not directly relevant to this PR but was curious how exactly the `numaNodeInfo` passed here and used at https://github.com/kubernetes/kubernetes/blob/d09f8b9d546abd7a7d29c81155571266f5df437e/pkg/kubelet/cm/devicemanager/topology_hints.go#L30 gets used/exercised in an e2e fashion.

Reply:
would like to pass nil but the function declaration doesn't use a pointer. I wait the CICD to get the exe